app/src/models/area.modelV2.js

```diff
@@ -38,6 +38,7 @@ const Area = new Schema({
     public: { type: Boolean, trim: true, required: true, default: false },
     fireAlerts: { type: Boolean, trim: true, required: true, default: false },
     deforestationAlerts: { type: Boolean, trim: true, required: true, default: false },
+    webhookUrl: { type: String, trim: true, required: true, default: false },
```

Comment:
Not required. Not sure what trim does.

Reply:
if you accidentally put ` https://my-url.io ` it will strip the ends